libbeat/metric/system/process/process.go

```diff
@@ -369,9 +368,11 @@ func (procStats *Stats) getProcessEvent(process *Process) common.MapStr {
 	}
 
 	if procStats.EnableCgroups {
-		if statsMap := cgroupStatsToMap(process); statsMap != nil {
-			proc["cgroup"] = statsMap
+		statsMap, err := process.RawStats.Format()
+		if err != nil {
+			procStats.logger.Warnf("Getting memory details: %v", err)
 		}
+		proc["cgroup"] = statsMap
```

Comment:
Should we use this statsMap if there was an error?
```suggestion
			procStats.logger.Warnf("Getting memory details: %v", err)
		} else {
		        proc["cgroup"] = statsMap
		}
```